sources/packages/web/src/components/common/sin/AddNewSIN.vue

```diff
@@ -0,0 +1,143 @@
+<template>
+  <v-form ref="addNewSINForm">
+    <modal-dialog-base
+      title="Add a new SIN"
+      :showDialog="showDialog"
+      max-width="730"
+    >
+      <template #content>
+        <error-summary :errors="addNewSINForm.errors" />
+        <div class="pb-2">
```

Comment:
This is a 20px padding. it mist be pb-5. or u can use py-5 to ensure on top and bottom
![image](https://user-images.githubusercontent.com/54600590/193693518-20237d6d-ddd8-4d0d-819a-ada935bdaaa2.png)